src/main/java/mate/academy/spring/service/UserServiceImpl.java

```diff
@@ -3,12 +3,14 @@
 import java.util.List;
 import mate.academy.spring.dao.UserDao;
 import mate.academy.spring.model.User;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 @Service
 public class UserServiceImpl implements UserService {
     private final UserDao userDao;
 
+    @Autowired
```

Comment:
We need to use @Autowired  annotation everywhere or nowhere to make our code more consistent. Remove it everywhere or add everywhere